packages/cursorless-vscode/src/keyboard/grammar/getAcceptableTokenTypes.ts

```diff
@@ -43,39 +66,80 @@ function getMetadata<T extends keyof KeyboardCommandHandler>(
     .metadata;
 }
 
+// Prevent infinite recursion
+const MAX_VISITS = 3;
+
+/** Indicates that the given symbol hasn't been typed yet */
+export const MISSING = Symbol("missing");
+
+/** The next token to be consumed */
```

Comment:
terminology/consistency: here you call it a token but three lines above you call it a symbol